src/client.ts

```diff
@@ -772,6 +816,48 @@ export class Client {
     bindingKeys.map((bk) => partitions.push(`${streamName}-${bk}`))
     return { partitions, streamBindingKeys: bindingKeys }
   }
+  private incrRefCount() {
+    return ++this.refCount
+  }
+
+  private decrRefCount() {
+    return --this.refCount
```

Comment:
same here

Reply:
this one is used when `close` is invoked to decide if the cleanup process must take place or not